src/main/java/uk/gov/hmcts/reform/userprofileapi/exception/ForbiddenException.java

```diff
@@ -0,0 +1,8 @@
+package uk.gov.hmcts.reform.userprofileapi.exception;
+
+public class ForbiddenException extends RuntimeException {
+
+    public ForbiddenException(String message) {
```

Comment:
why we need another exception?

Reply:
It is used in `FeatureConditionEvaluation` class which I have taken from PRD/CRD for LaunchDarkly